ansible_collections/arista/cvp/molecule/cv_image_v3/test_cv_image_v3.yaml

```diff
@@ -0,0 +1,138 @@
+---
+- name: Test cv_image_v3
+  hosts: CloudVision
+  connection: local
+  gather_facts: no
+  vars:
+    # Make sure given image does not exist on CVP
+    image_name: "EOS-4.25.4M.swi"
+  tasks:
+
+  ### Run all the tasks sequentially ###
+
+    - name: "Create bundle {{inventory_hostname}}"
+      vars:
+        ansible_command_timeout: 1200
+        ansible_connect_timeout: 600
+      arista.cvp.cv_image_v3:
+        mode: bundle
+        action: add
+        bundle_name: Test_bundle
+        image_list:
+          - "{{image_name}}"
+      register: CREATE_BUNDLE_DATA
+    - assert:
+        that:
+          - CREATE_BUNDLE_DATA.changed == true
+          - CREATE_BUNDLE_DATA.data.data == "Bundle : Test_bundle successfully created"
+          - CREATE_BUNDLE_DATA.failed == false
+
+    - name: "Update bundle {{inventory_hostname}}"
```

Comment:
what are we updating here?

Reply:
Here we are trying to check how "action: add" response when bundle is already present.
Currently we are updating using the same image, but if we have different image available in the same path we can modify the testcase.